src/main/proto/common.proto

```diff
@@ -66,6 +66,12 @@ enum ProcessingKey {
 
     // do not reuse reserved number to ensure forward and backward compatibility
     reserved 4, 5, 6;
+
+    /* key indicating whether Axon Server supports streaming. */
+    AS_SUPPORTS_STREAMING = 7;
```

Comment:
Why not call this `SERVER_SUPPORTS_STREAMING`? That feels a little more in line with `CLIENT_SUPPORTS_STREAMING`.

Reply:
Makes sense!